egs/chime6/s5b_track1/run.sh

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env bash
+#
+# Based mostly on the TED-LIUM and Switchboard recipe
```

Comment:
I think we should have a comment here on in a README about what "track 1" is, people in the future or who don't know about the eval may find this confusing.

Reply:
Thank you, I added the details.